README.rst

```diff
@@ -131,15 +131,17 @@ To work with the quipucords code, begin by cloning the repository::
 
     git clone git@github.com:quipucords/quipucords.git
 
-quipucords currently supports Python 3.5 and 3.6. If you do not have Python on your system, follow these `instructions <https://www.python.org/downloads/>`_. Based on the configuration of your system, you might be using either `pip` or `pip3` to install modules. The following instructions show the steps for a system with `pip`.
+quipucords currently supports Python 3.4, 3.5 and 3.6. If you do not have Python on your system, follow these `instructions <https://www.python.org/downloads/>`_. Based on the configuration of your system, you might be using either `pip` or `pip3` to install modules. The following instructions show the steps for a system with `pip`.
 
 
 Setting Up a Virtual Environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Add desired environment variables to the `.env` file.  You can copy ``.env.example`` to get started.
+
 You might want to isolate your development work by using a virtual environment. Run the following command to set up a virtual environment::
```

Comment:
Consider changing these directions to match the ones on Yupana: 
```
Developing inside a virtual environment is recommended. A Pipfile is provided. Pipenv is recommended for combining virtual environment (virtualenv) and dependency management (pip). To install pipenv, use pip

pip3 install pipenv
Then project dependencies and a virtual environment can be created using

pipenv install --dev
To activate the virtual environment run

pipenv shell
```

Reply:
These instructions don’t match qpc. We don’t have a Pipfile. We have a requirements file.